### ISSUE TITLE
Allow to use host sequences in hosts.py

### ISSUE THREAD

**Issue by lig** [lig/pyventory#8]
Detect lists of hosts in locals and put those hosts in inventory as well.
This should allow to the following:
```python
host_list = []
for n in range 5:
    host_list.append(MyAsset(num=n+1))
```

**Comment by lig** [comment]
```python
host_list = MyAsset() * 5
```
Looks nice also. However, it is not clear how to provide an iteration var to resulting instances.

**Comment by lig** [comment]
Fixed in #23
